packages/broker/src/ConfigWizard.ts

```diff
@@ -53,6 +32,21 @@ const generateApiKey = (): string => {
     return Buffer.from(hex).toString('base64').replace(/[^0-9a-z]/gi, '')
 }
 
+const DEFAULT_WS_PORT = WebsocketConfigSchema.properties.port.default
```

Comment:
These 4 temporary variables are not needed? Each component could access the info via DEFAULT_CONFIG_PORTS